src/System.IO/tests/StringWriter/StringWriterTests.cs

```diff
@@ -19,26 +19,27 @@ public class StringWriterTests
 
         private static char[] getCharArray()
         {
-            return new char[]{
-            char.MinValue
-            ,char.MaxValue
-            ,'\t'
-            ,' '
-            ,'$'
-            ,'@'
-            ,'#'
-            ,'\0'
-            ,'\v'
-            ,'\''
-            ,'\u3190'
-            ,'\uC3A0'
-            ,'A'
-            ,'5'
-            ,'\uFE70' 
-            ,'-'
-            ,';'
-            ,'\u00E6'
-        };
+            return new char[]
+            {
+                char.MinValue,
+                char.MaxValue,
+                '\t',
+                ' ',
+                '$',
+                '@',
+                '#',
+                '\0',
+                '\v',
+                '\'',
+                '\u3190',
+                '\uC3A0',
+                'A',
+                '5',
+                '\uFE70',
+                '-',
+                ';',
+                '\u00E6'
+            };
```

Comment:
Is there any change to these three files other than formatting?  Assuming it's just formatting, please revert the changes to those files.

Reply:
I asked him to fix the commas

Reply:
In that case, if we're going to modify it, why not consolidate the now four copies of this?

Reply:
Oh sorry. Looks like I missed this comment. Let me check...

Reply:
Actually there are two sets of the test data...
1. One in StreamReaderTests.cs which I copied over to TextReaderTests.cs and TextWriterTests.cs and,
1. Other one in StreamWriter.WriteTests.cs with a copy in StringWriterTests.cs

The second set uses fewer characters than the first set. Do we want to consolidate the copies which might entail re-writing some test cases? Given that it is test data how allowing each test to use its data? Just a thought...

Also I'm ok reverting back the formatting for the other places that I touched...

Inputs please.

Reply:
@stephentoub Bumping the thread. Please see above comment.

In addition, I made changes to StreamWriter.WriteTests.cs (method `setupArray`) and StringWriterTests.cs (method `getCharArray`) to attempt common input and tests ran fine. However for actual consolidation to single copy, it would entail restructuring the five test classes and extract data generation to a common base or test data provider class, also considering the fact that there are other smaller character data arrays which may differ across the five test files.

Appreciate your inputs.

Reply:
If there are two distinct sets of data, it's fine for them to remain separate.  My objection is to making copies of entire sets of data, and more generally to having exact duplicates... we should put the shared set or sets into some shareable place and use those shared sets rather than copy and pasting.

Reply:
@stephentoub I have tried to consolidate the data into a new `TestDataProvider` class. Please note that as mentioned in previous comment, even if there are two sets of data that subtly differ, I have consolidated them because with the longer of the two, the tests still passed. That was local testing. I will monitor the CI tests. Please check and let me know if the changes are acceptable.

Reply:
Also I have corrected the coding style issues that VS 2017 community edition IDE pointed out to me...